x/ethbridge/module.go

```diff
@@ -74,23 +75,25 @@ type AppModule struct {
 	AppModuleBasic
 	AppModuleSimulation
 
-	OracleKeeper oracle.Keeper
-	BankKeeper   bank.Keeper
-	Codespace    sdk.CodespaceType
-	Codec        *codec.Codec
+	OracleKeeper  oracle.Keeper
+	SupplyKeeper  supply.Keeper
+	AccountKeeper auth.AccountKeeper
+	Codespace     sdk.CodespaceType
+	Codec         *codec.Codec
 }
 
 // NewAppModule creates a new AppModule object
-func NewAppModule(oracleKeeper oracle.Keeper, bankKeeper bank.Keeper, codespace sdk.CodespaceType, cdc *codec.Codec) AppModule {
+func NewAppModule(oracleKeeper oracle.Keeper, supplyKeeper supply.Keeper, accountKeeper auth.AccountKeeper, codespace sdk.CodespaceType, cdc *codec.Codec) AppModule {
```

Comment:
we could clean up the params by introducing a keeper on this module

Reply:
will refactor to use expected_keepers and to create a bridge keeper in follow up PR